styles/stylesheet.css

```diff
@@ -0,0 +1,862 @@
+* {
+  box-sizing: border-box;
+}
+
+.hidden {
+  position: relative;
+  left: -10000px;
+  top: auto;
+  color: #484848;
+}
+
+.hidden:focus {
+  left: 0;
+}
+
+body {
+  color: #484848;
+  font-family: 'Work Sans', sans-serif;
+  font-size: 1rem;
+}
+
+.container {
+  display: grid;
+  grid-gap: 2rem;
+  grid-template-columns: auto;
```

Comment:
While this works well, we prefer you to use @media queries to manually set the grid columns for different screen sizes.

Reply:
Disregard my comment, I see you are using @media queries! Great work